### PR TITLE
feat: added playlist

### PR DESCRIPTION
**Pull request by ianuragsharma**
features added
Add playlist
remove playlist
add videos to playlist
remove videos from playlist

### CODE REVIEW

File: src/pages/playlist/PlaylistPage.jsx [hunk 1]
```diff
@@ -1,11 +1,20 @@
 import React from "react";
-import { Navbar } from "../../components";
+import { Navbar, PlaylistCard } from "../../components";
+import { useVideo } from "../../context";
 
 const PlaylistPage = () => {
+  const { videoState, videoDispatch } = useVideo();
+  const { playlists } = videoState;
+  const allPlaylist = playlists.map((playlist) => (
+    <PlaylistCard playlist={playlist}></PlaylistCard>
```
Comment (rahulyadav139) [inline src/pages/playlist/PlaylistPage.jsx:9]: Suggestion: it seems you missed to write 'key' attribute.

File: src/pages/singlePlaylistPage/SinglePlaylistPage.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  );
+  const allVideos = currentPlaylist[0].videos;
+  return (
+    <div>
+      <Navbar />
+      <div className="continer">
+        {allVideos.length > 0 ? (
+          <div className="flex-row  ml-1 videos-container">
+            {allVideos.map((video) => (
+              <ActionVideoCard video={video} />
```
Comment (rahulyadav139) [inline src/pages/singlePlaylistPage/SinglePlaylistPage.jsx:21]: here you also missed to write 'key' attribute.

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: Everything is looking great. Codes are readable. Great work.

File: src/pages/singlePlaylistPage/SinglePlaylistPage.jsx (hunk 2 above)
Comment (ianuragsharma) [reply src/pages/singlePlaylistPage/SinglePlaylistPage.jsx:21]: yes i missed it thanks for pointing it out

**Pull request merged**